README.md

```diff
@@ -17,84 +17,43 @@ limitations under the License.
 
 
 # <img src="logos/logo.svg" width="180">
+###A Tool for Complex and Scalable Data Access Policy Enforcement
 
-### Scalable Data Access Policy Management and Enforcement
+Palisade aims to provide a stable framework for defining and enforcing policies for accessing data.
 
-## Status
-<span style="color:red">
-Palisade is still in the early stages of development.
-At the moment there are a few simple examples to show how Palisade could work - it is not production ready.
-The description of Palisade below indicate the high level goals of the framework. 
-</span>
-
-## Documentation
-
-The documentation for the latest release can be found [here](https://gchq.github.io/Palisade).
 
+####Introduction
```

Comment:
Needs a space between #### and Introduction... see github render of this markdown

Reply:
Added the space.